src/exa.ts

```diff
@@ -252,6 +252,10 @@ const completionSpec: Fig.Spec = {
       description: "Shows icons before each folder or file",
       name: "--icons",
     },
+    {
+      description: "Displays when a file or folder got changed instead of modified in the header",
```

Comment:
It was also prettier complaining and not only eslint :)
```suggestion
      description: 
        "Displays when a file or folder got changed instead of modified in the header",
```